sim/hunter/runes.go

```diff
@@ -212,6 +221,17 @@ func (hunter *Hunter) applyCobraStrikes() {
 			}
 		},
 	})
+
+	hunter.RegisterAura(core.Aura{
+		OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+			if spell.Flags.Matches(SpellFlagShot) || spell.Flags.Matches(SpellFlagStrike) {
```

Comment:
I notice that the rune description mentions mongoose bite as well but I don't see it here

Reply:
I must've been asleep writing some of this lol